src/components/HamburgerMenu.vue

```diff
@@ -0,0 +1,24 @@
+<template>
+  <div class="menu-options">
```

Comment:
I suggest we don't use generic <div> tags, instead use <main> or <nav> or something like that instead.